pkgs/applications/networking/instant-messengers/riot/riot-desktop.nix

```diff
@@ -8,12 +8,12 @@
 
 let
   executableName = "riot-desktop";
-  version = "1.6.7";
+  version = "1.6.8";
   src = fetchFromGitHub {
     owner = "vector-im";
     repo = "riot-desktop";
     rev = "v${version}";
-    sha256 = "0l1ih7rkb0nnc79607kkg0k69j9kwqrczhgkqzsmvqxjz7pk9kgn";
+    sha256 = "1igln0a8inxamknsds524k8k7934frc7jjn6y9bn43d0mq2j2vh4";
   };
   electron = electron_7;
```

Comment:
Riot uses Electron 9 since v1.6.8-rc1 (changelog: https://github.com/vector-im/riot-desktop/blob/develop/CHANGELOG.md#changes-in-168-rc1-2020-07-01).

Reply:
It works with Electron 9 but mesa complains about missing iris drivers and there is no tray icon. But this might only be a problem on 20.03 and not unstable.

Reply:
The openGL driver is a constant source of troubles when testing new updates, because of the /run/opengl-driver impurity. This should probably be tested by rebuilding the system at this PR revision and boot into it, or in a VM.